js/signin.js

```diff
@@ -0,0 +1,114 @@
+const QUERY_SELECTOR = function (select) {
+  return document.querySelector(select);
+};
```

Comment:
먼저 반복적으로 사용되는 document.querySelector를 별도의 함수로 분리해주신 시도는 매우 좋은데요!
개선해보면 좋을 점이 있을 것 같아요!

1. 우선 `const` 키워드를 사용한다고 해도 함수를 선언하는 경우에는 대문자로 사용하지 않아요~ 따라서 아래와 같이 작성되어야 해요.
```suggestion
const querySelector = function (select) {
  return document.querySelector(select);
};
```

Reply:
2. 현재 함수는 함수 표현식으로 변수에 할당이 되어있는데요~ 이러한 경우에는 최근에는 거의 무조건 화살표 함수를 사용해주는게 일반적이에요~ 따라서 아래와 같이 작성이 될 수 있어요!
```suggestion
const querySelector = (select) => {
  return document.querySelector(select);
};
```

Reply:
3. (제 취향) 이런식으로 DOM을 가져오는 함수를 jQuery라는 라이브러리에서는 $와 같이 사용하는데요~! 이런 느낌도 괜찮을 것 같아요!
   
```suggestion
const $ = (select) => {
  return document.querySelector(select);
};
```

Reply:
4. 조금 더 확장성있게 사용한다면 DOM을 어디부터 찾기 시작할 것인지도 외부에서 전달받아서 사용할 수도 있을 것 같아요! 사소하시면 `select`보다 `selector`라는 이름이 더 적절할 것 같네요!
아! 그리구 화살표 함수로 사용한다면 코드를 조금 더 줄일 수 있을 것 같아요!

```suggestion
const $ = (selector, element = document) => element.querySelector(selector);
```